### ISSUE TITLE
possibility for tokenizing files

### ISSUE THREAD

**Issue by verenablaschke** [ismla-japanese-helper/japanese-helper#33]
maybe as a branch so we can potentially re-use it in the future?

**Comment by verenablaschke** [comment]
Added a branch: [`tokenize-files`](https://github.com/ismla-japanese-helper/japanese-helper/tree/tokenize-files). 

It's working for the sample file I added. I'll run it on the opensubtitles & tatoeba files later and add a note on how to use it in case we want to re-use it at some other point.

**Comment by verenablaschke** [comment]
I added a usage note to the README file in the branch.

I implemented this in a very quick way, by changing the other files and the general structure as little as possible. (The user files are turned into input streams when starting the program. When the user clicks `Tokenize files`, a single call to the server is made, where the files are then processed (file by file, line by line).)

On its own, I would say this all works reasonably fast--just under 15k lines/hr (~250 lines/min). The input files are pretty large though (180k lines for tatoeba, 2 million lines for osc), which I didn't realize until after it had already taken my computer over an hour to process less than 1% of the osc file...
Can we somehow speed this up? Or is this just a matter of picking a powerful computer where this can run over the course of some (long) weekend?
